projects/driver_controls/src/mech_brake.c

```diff
@@ -0,0 +1,78 @@
+
+
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <stdbool.h>
+#include "ads1015.h"
+#include "delay.h"
+#include "event_arbiter.h"
+#include "event_queue.h"
+#include "gpio_it.h"
+
+#include "input_event.h"
+#include "log.h"
+#include "mech_brake.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "wait.h"
+
+// takes in LSB as input and converts it to percentage using a linear relationship
+static int16_t lsb_to_percentage_converter(MechBrakeStorage *storage) {
+  int16_t percentage;
+
+  if (storage->calibration_data->zero_value > storage->calibration_data->hundred_value) {
```

Comment:
I feel like this could be simpler. You're only actually changing one variable in these calculations.